packages/@okta/vuepress-site/docs/reference/registration-hook/index.md

```diff
@@ -264,7 +264,59 @@ See [request properties](#objects-in-the-request-from-okta) for full details.
 }
 ```
 
-## Sample Progressive Enrollment request
+## Sample SSR responses
+
+The external service responds to Okta indicating whether to accept the end user's self-registration attempt. The response returns a `commands` object in the body of the HTTPS response. This object contains specific syntax that indicates whether the user is allowed or denied to self-register.
+
+See [response properties](#response-objects-that-you-send) for full details.
+
+### Sample SSR response using data.userProfile
+
+The following sample response shows a successful self-registration using a valid email domain:
+
+```json
+{
+    "commands": [
+        {
+            "type": "com.okta.user.profile.update",
+            "value": {
+                "login": "first.last@example.com"
+            }
+        }
+    ]
+}
+```
+
+### Sample SSR response using data.action
+
+The following sample response uses the `DENY` value and a custom error caused by an invalid email domain:
```

Comment:
In this case it's accurate based on what you see in the error object. But, in theory, all sorts of reasons you might want to deny someone. That said, I think it's good to display the error object here as it's the way we pass back details to the UI.